ocaml/idl/datamodel.ml

```diff
@@ -663,6 +663,8 @@ let _ =
     ~doc:"The specified VBD device is not recognized: please use a non-negative integer" ();
   error Api_errors.vm_not_resident_here [ "vm"; "host" ]
     ~doc:"The specified VM is not currently resident on the specified host." ();
+  error Api_errors.vm_not_resident [ "vm" ]
+    ~doc:"The specified VM is not currently resident on any host." ();
```

Comment:
Neither is wrong. I don't think the difference is particularly important.
Does "currently" add anything useful anyway?
If we do use the word, I think the least clumsy construction is to put it at the beginning or end of the sentence:
"Currently the specified VM is not resident on any host." or
"The specified VM is not resident on any host currently." (...or "at present" instead of "currently"...)

Is there any benefit to using the same pattern for the texts for `vm_not_resident_here` and `vm_not_resident`? Probably not.